content/millennium/r4/care-provision/care-plan.md

```diff
@@ -0,0 +1,146 @@
+---
+title: CarePlan | R4 API
+---
+
+# CarePlan
+
+* TOC
+{:toc}
+
+## Overview
+
+The R4 CarePlan resource can be used to represent care plans.
+
+### Care Plans
+
+The CarePlan resource describes the intentions of how one or more practitioners plan to deliver care to a specific patient. Care plans are used in a variety of areas and scopes ranging from encounter or visit specific to cross-visits.  Care plans are used to group activities, goals and/or practitioners to create context. Care plans are intended to be specific to the patient. It is possible for a planned activity to stand alone without a care plan such as a scheduled appointment.  
+
+The following fields are returned if valued:
+
+* [CarePlan Id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.status){:target="_blank"}
+* [Intent (plan)](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.intent){:target="_blank"}
+* [Category (assess-plan)](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.category){:target="_blank"}
+* [Title](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.title){:target="_blank"}
+* [Description](http://hl7.org/fhir/R4/careplan-definitions.html#CarePlan.description) {:target="_blank"}
+* [Subject (Patient Only)](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.subject){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.encounter){:target="_blank"}
+* [Effective Period](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.period){:target="_blank"}
+* [Author](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.author){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_plan, :r4) %>
+
+
+## Search
+
+Search for CarePlans that meet supplied query parameters:
+
+    GET /CarePlan?:parameters
+
+_Implementation Notes_
+
+* The [CarePlan.activity] element is not supported and will not be returned. This means that the [CarePlan.activity.detail.status] and [CarePlan.activity.detail.prohibited] modifier elements will not be returned.
+
+* The description of the CarePlan is included in the [CarePlan.text.div] element as ASCII text.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|--------------
+ `_id`        | See notes | [`token`]     | The logical resource id associated with the resource.
+ `date`       | N         | [`date`]      | Time period plan covers. Example: `date=ge2016&date=le2017 (Jan 1, 2016 - Dec 31, 2017)` or `date=ge2016-08-24T12:00:00.000Z&date=le2017-01-24T12:00:00.000Z`
+ `patient`    | See notes | [`reference`] | Who care plan is for. Example: `patient=1316024`
+ `category`   | See notes | [`token`]     | Only support 'assess-plan'
```

Comment:
Do we want to add description to explain how to utilize this parameter. The note should already cover that we only support assess-plan. Thoughts. 

Checkout DSTU2 for example: https://fhir.cerner.com/millennium/dstu2/care-provision/care-plan/

Reply:
Updated with example